### DISCUSSION TITLE
My search bar is not showing

### DISCUSSION THREAD

**Discussion by AntoniosBarotsis** [adityatelange/hugo-PaperMod#493] [Q&A]
Whenever I am running the website locally or generating the static minified HTML files, the search page is always rendered properly. On the hosted version however which is on Github pages so parts of the HTML file are missing including the search bar.

I have followed the search feature instructions [here](https://github.com/adityatelange/hugo-PaperMod/wiki/Features#search-page) as well as the FAQ [here](https://github.com/adityatelange/hugo-PaperMod/wiki/FAQs#search-not-working-) (although I'm not entirely sure if I should).

I can't figure out where the problem lies exactly so I'll just link the [repo](https://github.com/AntoniosBarotsis/Blog).

I am guessing that the issue might be in the [workflow](https://github.com/AntoniosBarotsis/Blog/blob/master/.github/workflows/deploy_bookdown.yml) I have set up but I am not sure.

You can find the search file [here](https://github.com/AntoniosBarotsis/Blog/blob/gh-pages/search/index.html), there's a `terms-tags` class at the end where the searchbox should be.

Any help is appreciated!

**Comment by pixelfabs** [comment]
I think this one is solved. Just checked the website and search works as intended.

**Reply by AntoniosBarotsis** [reply]
No, the fix was me force pasting the HTML file I had locally during deployment, it was still not working
